srcs/raycasting/dda_rays.c

```diff
@@ -99,6 +99,7 @@ void	ft_vrt_intersections(t_cub_data *data, t_dda *ray)
 
 void	ft_cast_ray(t_cub_data *data, t_dda *ray)
 {
+	printf("RAY ID[%d]\n", ray->id);
```

Comment:
You might want to remove these printf before merge ?

Also you have a WIP commit not squashed before merge.